chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -357,7 +359,23 @@
     --is-public True \
     --page-size 1 \
     2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
-[ -n "$DOCKER_IMAGE_ID" ] && echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+if [ -n "$DOCKER_IMAGE_ID" ] ; then
+  echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
+else
+  for node in #{docker_nodes.join(" ")}; do
+      ssh $node docker pull cirros
```

Comment:
I'm not very familiar with how docker works, but wouldn't it be enough to download it from the admin node and upload it to the glance node? why does it need to be on the "docker nodes" ?